src/app/applications/applications-list/applications-table/applications-table.component.html

```diff
@@ -8,16 +36,34 @@
     </div>
 
     <table mat-table [dataSource]="data" class="example-table" matSort matSortActive="name" matSortDisableClear
-      matSortDirection="asc">
+           matSortDirection="asc">
 
       <!-- Name Column -->
```

Comment:
Either this comment should be removed, or else it should be put in every column for the specific title :D